types/data.d.ts

```diff
@@ -32,7 +32,9 @@ export const enum Event {
     Performance = 25,
     ScriptError = 26,
     ImageError = 27,
-    LayoutSummary = 28
+    Layout = 28,
+    Crawl = 29,
```

Comment:
>Crawl [](start = 4, length = 5)

Crawl is the server action, not an event. Maybe RequiredResource?

Reply:
I will call it Resource.